api/src/services/script-runner.service.ts

```diff
@@ -195,6 +195,53 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @param jurisdiction should contain jurisdiction id
+   * @returns successDTO
+   * @description adds lottery released translations to the database
+   */
+  async addLotteryReleasedTranslations(
```

Comment:
This solution works and it follows our current practices for the script runner. I'm wondering if we will run into annoyances having to write a new script every time we need to add new translations. And then on top of that having to remember to run the script.

This is just an option and I'm open to the team discussing this as a solution but what do you think of this idea:

- Have the translations for each language and jurisdiction along with a default one as well in a json file. So for example for core it would be what is in the translation-factory but just pulled out to a separate file.
  - As a note, then we can pull that file directly into the translation-factory so it always has the up to date translations.
- Create a script runner endpoint that takes in an optional jurisdiction name. Then in this file have the service find the appropriate file and fully update the translations for either the default translations or the passed in jurisdiction. 
- Then whenever we want to add new or update existing translations we just update those files and run the script runner.

This would require the script to be run more than once but we would just handle that by having the script name be something like `update translations ${jurisdictions} <new date>`